.travis.yml

```diff
@@ -1,5 +1,34 @@
 sudo: required
 dist: trusty
+
+env:
+  COMPOSE_VERSION: 1.24.1
+  DSPACE_REST_HOST: localhost
```

Comment:
Because these `DSPACE*` environment variables are not referenced again in this Travis script, we might want a brief comment here to explain what they are doing.  Something as simple as `# Override default REST configuration to use  Docker instance (see `src/config.ts`)`.

It might just be me, but it looks confusing how these are being used.  I understand it now (after Slack discussion), but I wouldn't have understood it otherwise.